### DISCUSSION TITLE
/gongju/choco.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#75] [Announcements]
# /gongju/choco.html

chocolatey：一款 GitHub 星标 8.2k+ 的 Windows 命令行软件管理神器，好用到爆！

https://tobebetterjavaer.com/gongju/choco.html

**Comment by ibacon66** [comment]
装了，用了，雀氏润

**Comment by ibacon66** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
